k8s/base/app/invoice-cronjob.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: invoice-cronjob
+spec:
+  schedule: "0 13 1 * *" # TODO Is this the correct schedule?
```

Comment:
Is this the cronjob schedule we want?

Reply:
I think we can run this more often than just once a month. For example: every week + the first of every month.